src/tooltip/css/objects/_honeycomb.tooltip.objects.tooltip.scss

```diff
@@ -0,0 +1,96 @@
+
+/* Base styles for the entire tooltip and default position*/
+.tooltip, [data-tooltip], .tooltip-top {
+  position: relative;
+  cursor: help;
+
+  /* Shared animations and properties */
+  &:before, &:after {
+    position: absolute;
+    visibility: hidden;
+    -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=0)";
+    filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=0);
+    opacity: 0;
+    -webkit-transition:
+            opacity 0.2s ease-in-out,
+            visibility 0.2s ease-in-out,
+            -webkit-transform 0.2s cubic-bezier(0.25, 0.1, 0.25 ,1);
+    -moz-transition:
+            opacity 0.2s ease-in-out,
+            visibility 0.2s ease-in-out,
+            -moz-transform 0.2s cubic-bezier(0.25, 0.1, 0.25 ,1);
+    transition:
+            opacity 0.2s ease-in-out,
+            visibility 0.2s ease-in-out,
+            transform 0.2s cubic-bezier(0.25, 0.1, 0.25 ,1);
+    -webkit-transform: translate3d(0, 0, 0);
+    -moz-transform:    translate3d(0, 0, 0);
+    transform:         translate3d(0, 0, 0);
```

Comment:
Looking into it a bit more, it might be that `-moz` prefixes aren't so needed any more? Just looking at `transform`, unprefixed support was added in FF 16 in 2012, and global usage below that is ~0.01% (I /think/ that our autoprefixer has some logic that it adds support for browsers with >1% global usage)

so I guess I'd just drop the prefixed rules, if you're happy to!